src/AutoCrudAdmin/Extensions/ControllerExtensions.cs

```diff
@@ -61,12 +61,12 @@ public static class ControllerExtensions
 
         if (typeof(TEntityId) == typeof(string) && !string.IsNullOrWhiteSpace(idAsString))
         {
-            id = idAsString;
+            id = idAsString.Trim();
             isSuccessful = true;
         }
         else if (int.TryParse(idAsString, out var entityIdInt))
         {
-            id = entityIdInt;
+            id = entityIdInt.ToString().Trim();
```

Comment:
How can entityIdInt have empty spaces? Isn't supposed to be integer?

Reply:
I added it for an additional check. Removing it.